2-structured-data/src/DataModel/CloudSql.php

```diff
@@ -53,7 +53,18 @@ public function __construct($dsn, $user, $password)
             return explode(' ', $columnDefinition)[0];
         }, $columns);
         $columnText = implode(', ', $columns);
-        $pdo = $this->newConnection();
+
+        try {
+            $pdo = $this->newConnection();
+        } catch (\PDOException $e) {
+            if (false === strpos($e->getMessage(), 'Unknown database')) {
```

Comment:
No, if we tell the users how to create the DB in the tutorial using the cloudSQL proxy, this can be removed.